### DISCUSSION TITLE
汉化工具最近会漏字

### DISCUSSION THREAD

**Discussion by gin32167** [phantom-sea-limited/Crawler#16] [General]
### 是否有已经存在的问题

- [X] 我已经查阅了 https://github.com/phantom-sea-limited/Crawler/issues
- [X] 我正在使用最新版的爬虫

### 是什么爬虫引发的问题

sangtacviet.user.js

### 你在哪个网页遇到的问题

https://sangtacviet.com/truyen/ciweimao/1/100376063/110549592/

### 描述问题

sangtacviet的汉化工具，最近会漏字，同一章节的内容，以前汉化就未漏字

### 其他

<img width="898" alt="001" src="https://github.com/phantom-sea-limited/Crawler/assets/83170209/f200d84b-3460-4e53-abe6-d1081b278f85">
这是以前下载的章节
<img width="818" alt="002" src="https://github.com/phantom-sea-limited/Crawler/assets/83170209/309983df-23f9-40a7-aa13-d58759b3429d">

**Comment by Rcrwrate** [comment]
目前这个缺字的问题是明确sangtacviet方面的措施

他们似乎对原文进行了删改

![image](https://github.com/phantom-sea-limited/Crawler/assets/46920034/a07b262d-f839-473d-a44f-5598b237c146)

这个问题，非常遗憾**暂时无法修复**

不过，这个问题似乎只在刺猬猫出现，你可以使用nhimmeo的服务[zh.nhimmeo.cf](zh.nhimmeo.cf)

有人说STV做了个APP，这里面确实是完整的中文，[下载地址](https://staticvn.sangtacvietcdn.xyz/download/stvmobilereader-autoupdate-1.0.3.apk)

这个东西，我之前看过，要逆向得底层逆向，这得肉眼识别汇编，除非有巨佬下场，不然很难搞

![5a1a2f95a4c27d1edd0046f45dd5ad6eddc43886](https://github.com/phantom-sea-limited/Crawler/assets/46920034/000af491-8fd8-4f21-b617-b6883ea32cb6)

先容我观察一下

**Reply by liyiyi99** [reply]
大佬，软件怎么显示中文，找了半天没找着。

**Reply by Rcrwrate** [reply]
> 大佬，软件怎么显示中文，找了半天没找着。

我好久没用了，你摸索摸索，每个按钮都点点看

**Reply by GithubElbud** [reply]
> > 大佬，软件怎么显示中文，找了半天没找着。
> 
> 我好久没用了，你摸索摸索，每个按钮都点点看



> 大佬，软件怎么显示中文，找了半天没找着。



> > 大佬，软件怎么显示中文，找了半天没找着。
> 
> 我好久没用了，你摸索摸索，每个按钮都点点看

请问一下，为什么网站下面的显示原文没有有，但软件上去点一个个文字却能小段显示原文，是stv故意的吗

**Comment by 1227briankuo** [comment]
下載app測試，目前看起來刺蝟貓來源好像還是會缺字(https://sangtacviet.com/app.v2.php)
可能需要再確認是不是app也套用和web相同邏輯了

如果這樣的話，前端好像幾乎無解?
只能想到將h attr音譯的值反推回去，才能補全缺的字了，需要懂語言的巨佬

(不知道2月如果nhimmeo有導入stv是不是也會缺字)

**Reply by Rcrwrate** [reply]
APP，并且是中国IP，好像就没问题了（我当时遇到的抽象情况就是不挂VPN打不开，挂了看不了）

**Comment by Rcrwrate** [comment]
顺带一提cwm的话

https://trxs.cc/

姑且可以平替，这站长似乎是和刺猬猫过不去一样

**Comment by blibilijojo** [comment]
写了个stv第三方版，有兴趣的加QQ群：1087637832
